### ISSUE TITLE
Horrible result when I run the tutorial on the same video as the demo

### ISSUE THREAD

**Issue by mostafachatillon** [Russell91/TensorBox#106]
I ran the tutorial on the same video as the demo, and the result is horrible (in blue the boxes from the original demo, in green my boxes):
https://www.youtube.com/watch?v=415jvF8aELs&feature=youtu.be

I don't understand why.

My exact code is:

`  GNU nano 2.2.6                                                                                                                                                                                           File: create_video.py                                                                                                                                                                                                                                                                                                                                                                                              

import os
import json
import subprocess
from scipy.misc import imread

from scipy import misc
#%matplotlib inline
import cv2 #attention a l'ordre d'importation. importer d'abord cv2

from train import build_forward
from utils import train_utils
from utils.annolist import AnnotationLib as al
from utils.stitch_wrapper import stitch_rects
from utils.train_utils import add_rectangles
from utils.rect import Rect
from utils.stitch_wrapper import stitch_rects
from evaluate import add_rectangles
import cv2


#cell 2

hypes_file = './hypes/overfeat_rezoom.json'
iteration = 150000
with open(hypes_file, 'r') as f:
    H = json.load(f)
true_json = './data/brainwash/val_boxes.json'
pred_json = './output/%d_val_%s.json' % (iteration, os.path.basename(hypes_file).replace('.json', ''))
true_annos = al.parse(true_json)

#cell 3



def create_video(filename):

  tf.reset_default_graph()
  x_in = tf.placeholder(tf.float32, name='x_in', shape=[H['image_height'], H['image_width'], 3])
  if H['use_rezoom']:
    pred_boxes, pred_logits, pred_confidences, pred_confs_deltas, pred_boxes_deltas = build_forward(H, tf.expand_dims(x_in, 0), 'test', reuse=None)
    grid_area = H['grid_height'] * H['grid_width']
    pred_confidences = tf.reshape(tf.nn.softmax(tf.reshape(pred_confs_deltas, [grid_area * H['rnn_len'], 2])), [grid_area, H['rnn_len'], 2])
    if H['reregress']:
        pred_boxes = pred_boxes + pred_boxes_deltas
  else:
    pred_boxes, pred_logits, pred_confidences = build_forward(H, tf.expand_dims(x_in, 0), 'test', reuse=None)

  saver = tf.train.Saver()



  with tf.Session() as sess:
    sess.run(tf.initialize_all_variables())
    saver.restore(sess, 'data/overfeat_rezoom/save.ckpt-%dv2' % iteration)

    annolist = al.AnnoList()
    import time; t = time.time()
#    for i in range(0, 500):
#        true_anno = true_annos[i]
#        img = imread('./data/brainwash/%s' % true_anno.imageName)



    cap = cv2.VideoCapture(filename)


    ret, frame = cap.read()
    height , width =  480, 640

#video = cv2.VideoWriter('baby_printed.avi',-1,1,(width,height))



#fourcc = cv2.VideoWriter_fourcc(*'XVID')
    fourcc = cv2.cv.CV_FOURCC(*'XVID')
    video = cv2.VideoWriter(filename[:-4]+'_printed.avi',fourcc, 25.0, (width,height))

    while(ret==True):
    # Capture frame-by-frame
      ret, frame = cap.read()

      if ret==True:


        img=cv2.resize(frame,(640,480))

        feed = {x_in: img}
        (np_pred_boxes, np_pred_confidences) = sess.run([pred_boxes, pred_confidences], feed_dict=feed)
        pred_anno = al.Annotation()
        pred_anno.imageName = file # true_anno.imageName
        new_img, rects = add_rectangles(H, [img], np_pred_confidences, np_pred_boxes,
                                        use_stitching=True, rnn_len=H['rnn_len'], min_conf=0.7,
                                        show_suppressed=False)

        pred_anno.rects = rects
        annolist.append(pred_anno)

        print('la shape de new image est : ' + str(new_img.shape))
        video.write(new_img)
        cv2.waitKey(20)
        if cv2.waitKey(1) & 0xFF == ord('q'):
            break

    else:
        print('we exit the loop')
# Release everything if job is finished
  cap.release()
  video.release()
  cv2.destroyAllWindows()`

**Comment by Russell91** [comment]
What do your precision recall curves in the notebook look like?
